src/osemosys.txt

```diff
@@ -562,14 +571,17 @@ s.t. SI10_TotalDiscountedCostByStorage{r in REGION, s in STORAGE, y in YEAR}: Di
 #########       	Capital Costs 		     	#############
 #
 s.t. CC1_UndiscountedCapitalInvestment{r in REGION, t in TECHNOLOGY, y in YEAR}: CapitalCost[r,t,y] * NewCapacity[r,t,y] = CapitalInvestment[r,t,y];
-s.t. CC2_DiscountingCapitalInvestment{r in REGION, t in TECHNOLOGY, y in YEAR}: CapitalInvestment[r,t,y]/(CapitalRecoveryFactor[r,t,y]) = DiscountedCapitalInvestment[r,t,y];
+
+s.t. CC2_DiscountingCapitalInvestment{r in REGION, t in TECHNOLOGY, y in YEAR: TypeOfDiscountRate[r]=1}: CapitalInvestment[r,t,y]/(CapitalRecoveryFactor[r,y]) = DiscountedCapitalInvestment[r,t,y];
+
+s.t. CC3_DiscountingCapitalInvestment{r in REGION, t in TECHNOLOGY, y in YEAR: TypeOfDiscountRate[r]=2}: CapitalInvestment[r,t,y]*(CapitalRecoveryFactorIdv[r,t,y]) = DiscountedCapitalInvestment[r,t,y];
```

Comment:
Do we need this extra equation, or could we just ensure that the `DiscountRateIdv` defaults to the `DiscountRate`? We could then also remove the `TypeOfDiscountRate` parameter.